assertj-android/src/main/java/org/assertj/android/api/widget/AbstractHorizontalScrollViewAssert.java

```diff
@@ -0,0 +1,55 @@
+// Copyright 2013 Square, Inc.
+package org.assertj.android.api.widget;
+
+import android.widget.HorizontalScrollView;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public abstract class AbstractHorizontalScrollViewAssert<S extends AbstractHorizontalScrollViewAssert<S, A>, A extends HorizontalScrollView>
+    extends AbstractFrameLayoutAssert<S, A> {
+  protected AbstractHorizontalScrollViewAssert(A actual, Class<S> selfType) {
+    super(actual, selfType);
+  }
+
+  public AbstractHorizontalScrollViewAssert hasMaximumScrollAmount(int amount) {
```

Comment:
These should return `S` and need to use the `self` trick (copy/paste from other abstract ones).